.github/workflows/ci.yml

```diff
@@ -246,7 +246,6 @@ jobs:
   build:
     name: Build
     if: ${{ !contains(github.event.pull_request.title, '[skip tests]') }}
-    needs: test-import
```

Comment:
I've removed this as the smoke tests are randomly failing. I think it is OK to run the smoke tests and build in parallel.